packages/fast-usdc/src/utils/fees.js

```diff
@@ -27,28 +29,38 @@ const { add, isGTE, subtract } = AmountMath;
  * }} RepayPaymentKWR
  */
 
-/** @param {FeeConfig} feeConfig */
-export const makeFeeTools = feeConfig => {
+/**
+ * @param {FeeConfig} feeConfig
+ * @param {Record<CaipChainId, FeeConfig>} [customFeeConfigs] fee configs scoped by `CaipChainId`
+ */
+export const makeFeeTools = (feeConfig, customFeeConfigs = harden({})) => {
```

Comment:
the conceptual distinction between "fee config" and "custom fee config" is awkward to me. They are all aspects of the fee _Iconfiguration_.

Please unify with a single `feeConfig`. What is currently in the `FeeConfigShape` could be called `FeeParameters` if you want to maintain that schema for each custom record.

```ts
export type FeeParams = {
  /** flat fee charged for every advance */
  flat: Amount<'nat'>;
  /** proportion of advance kept as a fee */
  variableRate: Ratio;
  /** proportion of fees that goes to the contract (remaining goes to LPs) */
  contractRate: Ratio;
};

export type FeeConfig = {
  base: FeeParams;
  custom: Record<string, FeeParams>;
};
```

Reply:
The above changes the vstorage schema incompatibly. I think it's worthwhile for clarity, as long as we update the frontend before this goes to Mainnet.

Alternately, we could have,
```ts
export type FeeConfig = FeeParams & {
  toChain?: Record<string, FeeParams>;
};
```

Reply:
Discussed offline, went with: 

```ts
export type FeeConfig = {
  /** flat fee charged for every advance, eligible for LP disbursement */
  flat: Amount<'nat'>;
  /** proportion of advance kept as a fee, eligible for LP disbursement */
  variableRate: Ratio;
  /** proportion of `flat` and `variableRate` fees that goes to the contract (remaining goes to LPs) */
  contractRate: Ratio;
  /**
   * if present, a fee required to relay (e.g. CCTP to EVM/Solana). Not
   * considered for LP disbursement - goes to `contractSeat`.
   */
  relay?: Amount<'nat'>;
  /** Optional destination-specific overrides */
  destinationOverrides?: Record<
    CaipChainId,
    Partial<Omit<FeeConfig, 'destinationOverrides'>>
  >;
};
```